Test/DependencyCollector/FunctionalTests/TestApps/Aspx451/ApplicationInsights.config

```diff
@@ -47,4 +37,10 @@
     <Add Type="Microsoft.ApplicationInsights.Web.SessionTelemetryInitializer, Microsoft.AI.Web"/>
     <Add Type="Microsoft.ApplicationInsights.WindowsServer.BuildInfoConfigComponentVersionTelemetryInitializer, Microsoft.AI.WindowsServer"/>
   </TelemetryInitializers>
+  <TelemetryChannel Type="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.ServerTelemetryChannel, Microsoft.AI.ServerTelemetryChannel"/>
```

Comment:
This will break the Func tests. (Ai.Config is destroyed after every nuget upgrade) Please revert this file to original.